lib/challenge.js

```diff
@@ -115,10 +132,15 @@ function prompt6() {
  */
 function prompt7() {
   // YOUR CODE HERE
+  for(i = 0; i <= 100; i++){
+    if(i % 5 == 0 && i > 1){
+    console.log(`I found a ${i}! High Five Baby!`);
+    }
+  }
```

Comment:
If you are able to I suggest using strict comparison `===` 

Later when we are getting into working with data it will help you keep the flow straight.

Reply:
stricts only from now on 🫡

Reply:
Sometimes loose comparisons can come in handy too. since it is not a strictly typed language JavaScript has some weird quirks you can get away with and sometimes use to your advantage. I'm mainly bringing it up to help you add a layer of intent to your code